src/cosmology/cosmology_functions.cpp

```diff
@@ -130,4 +174,64 @@ void Grid3D::Change_GAS_Frame_System(bool forward)
   }
 }
 
+/* local function that designates whether we are using a root-process. It gives
+ * gives a sensible result regardless of whether we are using MPI */
+static inline bool Is_Root_Proc()
+{
+  #ifdef MPI_CHOLLA
+  return procID == root;
+  #else
+  return true;
+  #endif
+}
+
+void Cosmology::Create_Expansion_History_File(struct Parameters *P)
+{
+  // writing the expansion history
```

Comment:
Minor suggestion for docstring-formatting

```suggestion
/* create the file for recording the expansion history */
void Cosmology::Create_Expansion_History_File(struct Parameters *P)
{
```

Reply:
I'll default to this format in the future.